plugins/modules/nmcli.py

```diff
@@ -64,6 +64,9 @@
             - Type V(infiniband) is added in community.general 2.0.0.
             - Type V(loopback) is added in community.general 8.1.0.
             - Type V(macvlan) is added in community.general 6.6.0.
+            - Type V(ovs-bridge) is added in community.general 8.5.0.
+            - Type V(ovs-interface) is added in community.general 8.5.0.
+            - Type V(ovs-port) is added in community.general 8.5.0.
```

Comment:
8.5.0 has already been released, the next release accepting new features is 8.6.0.
```suggestion
            - Type V(ovs-bridge) is added in community.general 8.6.0.
            - Type V(ovs-interface) is added in community.general 8.6.0.
            - Type V(ovs-port) is added in community.general 8.6.0.
```